master/master.cfg

```diff
@@ -236,6 +236,7 @@ builders = [
     ("AMD64 Fedora Rawhide Clang Installed", "cstratak-fedora-rawhide-x86_64", ClangUnixInstalledBuild, UNSTABLE),
     ("AMD64 Fedora Rawhide LTO", "cstratak-fedora-rawhide-x86_64", LTONonDebugUnixBuild, UNSTABLE),
     ("AMD64 Fedora Rawhide LTO + PGO", "cstratak-fedora-rawhide-x86_64", LTOPGONonDebugBuild, UNSTABLE),
+    ("x86 Arch Linux", "pablogsal-arch-x86_64", UnixBuild, UNSTABLE),
```

Comment:
`("AMD64 Arch Linux", "pablogsal-arch-x86_64", UnixBuild, UNSTABLE),`

We pretty much use "AMD64" everywhere, although we have a macOS builder using "x86-64" (I suppose because it's definitely an Intel processor, but that also doesn't really matter anyway :)).

Reply:
I have opened https://github.com/python/buildmaster-config/pull/127